js/main.js

```diff
@@ -1,2 +1,25 @@
 /* global data */
 /* exported data */
+
+var photoURL = document.querySelector('.photourl');
+var photo = document.querySelector('img');
+var form = document.querySelector('.form');
+var title = document.querySelector('.title');
+var notes = document.querySelector('.notes');
+
+photoURL.addEventListener('input', updateImage);
+form.addEventListener('submit', submit);
+
+function updateImage(event) {
+  photo.setAttribute('src', photoURL.value);
+}
+
+function submit(event) {
+  event.preventDefault();
+  var object = { title: title.value, url: photoURL.value, notes: notes.value };
+  object.nextEntryId = data.nextEntryId;
```

Comment:
This property name is a bit strange. On the `data` model, `nextEntryId` makes sense since it holds the next entry id to use, but in this case, `nextEntryId` is not a fitting name. This would be better as something like `entryId`